src/errors.rs

```diff
@@ -1,3 +1,23 @@
+/*
+ * Copyright 2021 Julian Schmidhuber <github@schmiddi.anonaddy.com>
+ *
+ * This file is part of Tubefeeder.
+ *
+ * Tubefeeder is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Tubefeeder is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Foobar.  If not, see <https://www.gnu.org/licenses/>.
```

Comment:
Foobar

Reply:
Should be fixed in the latest push. I believe you were too fast.